dev-client/src/screens/ProjectInputScreen/ProjectInputScreen.tsx

```diff
@@ -0,0 +1,157 @@
+  const soilSettings = useSelector(state =>
+    selectProjectSettings(state, projectId),
+  );
+  const dispatch = useDispatch();
+  const onInfoPress = useInfoPress();
+
+  const onEditInstructions = useCallback(() => {
+    return navigation.navigate('EDIT_PROJECT_INSTRUCTIONS', {project: project});
+  }, [navigation, project]);
+
+  const onProjectPrivacyChanged = useCallback(
+    (privacy: 'PRIVATE' | 'PUBLIC') =>
+      dispatch(updateProject({id: project!.id, privacy})),
+    [project, dispatch],
+  );
+
+  const onSave = () => {
+    // Manual save button is for reassurance, all items auto-save.
+    navigation.pop();
+  };
+
+  return (
+    <VStack height={'full'}>
+      <ScrollView>
+        <Box p={4} alignItems="flex-start">
+          <HStack pb={4}>
+            <RadioBlock
+              label={
+                <HStack>
+                  <Text variant="body1" bold>
+                    {t('site.dashboard.privacy')}
+                  </Text>
+                  <IconButton
+                    pt={0}
+                    pb={0}
+                    pl={2}
+                    size="md"
+                    name="info"
+                    onPress={onInfoPress}
+                    _icon={{color: 'action.active'}}
+                  />
+                </HStack>
+              }
+              options={{
+                PUBLIC: {text: t('privacy.public.title')},
+                PRIVATE: {text: t('privacy.private.title')},
+              }}
+              groupProps={{
+                name: 'project-privacy',
+                onChange: onProjectPrivacyChanged,
+                value: project.privacy,
+                ml: '',
+              }}
+            />
+          </HStack>
+          <Text bold fontSize={'md'}>
+            {t('projects.inputs.instructions.title')}
+          </Text>
+          <Text fontSize={'md'}>
+            {t('projects.inputs.instructions.description')}
+          </Text>
+          <Button
+            mt={2}
+            pl={4}
+            pr={4}
+            size="lg"
+            backgroundColor="primary.main"
+            shadow={5}
+            onPress={onEditInstructions}>
+            <HStack>
+              <Icon color="primary.contrast" size={'sm'} mr={2} name={'edit'} />
+              <Text color="primary.contrast" textTransform={'uppercase'}>
+                {t('projects.inputs.instructions.add_label')}
+              </Text>
+            </HStack>
+          </Button>
+        </Box>
+        <Accordion
+          Head={
+            <Text pt={3} pb={3} fontSize={'md'} color={'primary.contrast'}>
+              {t('soil.pit')}
+            </Text>
+          }>
+          <EnsureDataPresent
+            data={soilSettings}
+            Component={SoilPitSettings}
+            props={{projectId}}
+          />
+        </Accordion>
+        <Box height={4} />
+        <Accordion
+          Head={
+            <Text pt={3} pb={3} fontSize={'md'} color={'primary.contrast'}>
```

Comment:
question: Shouldn't these be bare strings? In various places.
```suggestion
            <Text pt={3} pb={3} fontSize="md" color="primary.contrast">
```

Reply:
I think that this is from some older code that I was moving around to different files in this PR. Ideally we would only pass string props as strings for consistency, but it functionally does not make a difference.